app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -48,6 +46,24 @@ def submission_status
 
     private
 
+    def submit_4142(form_content, claim)
+      if form_content['form4142'].present?
+        translated_form = EVSS::DisabilityCompensationForm::Form4142.new(@current_user, form_content).translate
+        CentralMail::SubmitForm4142Job.perform_async(
+          @current_user.uuid, translated_form, claim.id, claim.created_at
```

Comment:
I believe the claim id is the response.claim_id from the `SubmitForm526` job so this logic may have to move to that job. See https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12868#issuecomment-419315834